.circleci/config.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
       - restore_cache:
           keys:
-            - dep-cache-{{ checksum "Gopkg.toml" }}-{{ checksum "Gopkg.lock" }}
+            - dep-cache-{{ checksum "Gopkg.lock" }}
```

Comment:
not quite sure how the checksum calculation works - but Gopkg.lock order of dependencies is not deterministic.  i.e. each dependent repo may still have the same commit ID or tag, but the order of the dependencies is not consistent

Reply:
That is a good find @GregHanson !.. I wish you had caught this yesterday when I made this change!

Reply:
It doesn't really matter - the lock file is not touched by dep update -vendor-only.
And when the nightly or whatever updates dependencies - we can reset the cache ( likely the dep update will also generate the cache)